DoLahTest/ParserUnitTest.cpp

```diff
@@ -71,52 +106,35 @@ namespace DoLahTest {
 
     TEST_METHOD_INITIALIZE(Startup) {
         time_t t = time(0);
-        std::tm current;
         localtime_s(&current, &t);
```

Comment:
Please don't do this, mock the date instead. Have a fix boundary test input. Random variable is not good to be part of your test variable. (sometime it works, sometime it don't)